android.sh

```diff
@@ -826,14 +826,14 @@ if [[ ! -z ${ANDROID_ARCHITECTURES} ]]; then
 
     echo -e -n "\n\nCreating Android archive under prebuilt/android-aar: "
 
-    ./gradlew clean build 1>>${BASEDIR}/build.log 2>&1
+    ./gradlew clean app:assembleRelease app:testReleaseUnitTest 1>>${BASEDIR}/build.log 2>&1
```

Comment:
I tested this step and noticed that library is not obfuscated. Is there a specific reason for that?

Reply:
I guess it was always like this. It is stated [here](https://github.com/Javernaut/mobile-ffmpeg/blob/android_demo_improvement/android/app/build.gradle#L17). This line explicitly says 'do not obfuscate the code for the release variant'

Reply:
You are right. We'll need to set `minifyEnabled` to true.

Reply:
As you've said, this section builds both `:app` and `:test-app`. I remember that I wanted to build only `:app` at some point. But building `:test-app` also helped me to verify that `test-app` is up to date with the `:app` itself. That's why I kept it this way. But I think we don't need to do that here. `test-app` is not part of the actual build process.